tests/test_serialization.py

```diff
@@ -148,3 +150,16 @@ def test_secure_dump():
     assert_raises(PicklingError, secure_dump, bar, f.name)
     with open(f.name, 'rb') as f:
         assert type(load(f)) is object
+
+
+def test_dump_and_add_to_dump():
+    x = 3
+    y = 2
+    path = 'test.tar'
```

Comment:
Can you please use `NamedTemporaryFile` like we do e.g. [there](https://github.com/mila-udem/blocks/blob/master/tests/test_serialization.py#L135)? Good thing about temporary files is that you don't have care about their deletion that much. They will be automatically deleted at some point anyway.